### PR TITLE
MAR ipfs1.3 - Gateways

### PR DESCRIPTION
**Pull request by mrodriguez3313**
- Added gateway goals to json
- I moved content around in the original Gateway page
- I moved DNSlink info that was commented out from ipfs1.2 changes into gateways instead. This is relevant here because you use the [path-gateway URL schema](https://curriculum.pl-launchpad.io/curriculum/ipfs/ipfs-gateways/#path-gateway) to access data from DNSLink. And implementors may need this to create their own gateways. 
- I cut out large portions of the [current projects](https://curriculum.pl-launchpad.io/curriculum/ipfs/ipfs-gateways/#current-projects-1) section to link to their respective articles instead. Shortening this page by over 30 lines, all while adding in more relevant content.

### CODE REVIEW

File: content/en/curriculum/ipfs/ipfs-gateways.md [hunk 1]
```diff
@@ -52,131 +59,93 @@ Example:
...
 
-#### DNSLink
+### DNSLink
 
 [DNSLink](https://dnslink.dev) enables you to map a domain name to an IPFS address (CID or IPNS libp2p-key) by means of a DNS TXT record.
 
-- Read the [DNSLink guide](https://docs.ipfs.io/concepts/dnslink/) for details, including how to set it up on your own website
-- See details on [DNSLink in IPFS Companion](https://docs.ipfs.io/how-to/dnslink-companion/) to see additional benefits of using IPFS Companion with DNSLink support
+To create a mutable pointer with [DNSLink](https://docs.ipfs.io/concepts/dnslink/), you need:
+- a domain name you control
```
Comment (walkerlj0) [inline content/en/curriculum/ipfs/ipfs-gateways.md:110]:
```suggestion
- A domain name you control (e.g. `www.me.com`)
```

File: content/en/curriculum/ipfs/ipfs-gateways.md [hunk 2]
```diff
@@ -52,131 +59,93 @@ Example:
...
+- A CID or an IPNS name to link to
```
Comment (walkerlj0) [inline content/en/curriculum/ipfs/ipfs-gateways.md:111]:
```suggestion
- A CID or an IPNS name to link to, such as a website published with [Fleek.co](https://fleek.co/)
```

File: content/en/curriculum/ipfs/ipfs-gateways.md [hunk 3]
```diff
@@ -52,131 +59,93 @@ Example:
...
+#### Quick explanation of dnslink in IPFS
 
-#### [W3C](https://www.w3.org/Consortium/)
+{{< youtube YxKZFeDvcBs >}}
 
-Protocol Labs is a W3C Member. Current focus is to watch, learn, and participate in [WebExtensions Community Group](https://www.w3.org/community/webextensions/).
+Source: [Introduction to DNSlink](https://dnslink.dev/#introduction), [IPFS docs on DNSLink](https://docs.ipfs.tech/concepts/dnslink/)
 
-#### IPFS and Igalia collaborate on dweb in browsers
+### Collaborations
```
Comment (walkerlj0) [inline content/en/curriculum/ipfs/ipfs-gateways.md:125]:
```suggestion
## Collaborations
```

File: content/en/curriculum/ipfs/ipfs-gateways.md [hunk 4]
```diff
@@ -52,131 +59,93 @@ Example:
...
- - IPFS [libdweb experiments](https://github.com/ipfs-shipyard/ipfs-companion/blob/libdweb/docs/libdweb.md), including a native protocol handler, local DNS-SD discovery and TCP transport
- - The long-term goal of this project was to integrate these APIs into the WebExtensions ecosystem, but as of Q3 2020 it is not yet in Firefox Nightly
-- Exposing the IPFS API via `window.ipfs`(experiment ended in 2020)
-- Support for `chrome.sockets.*` APIs in Chromium browsers (deprioritized due to EOL 2022)
+[IPFS Companion](https://github.com/ipfs-shipyard/ipfs-companion#ipfs-companion) is a browser extension that simplifies access to IPFS resources and adds browser support for the IPFS protocol. It runs in <img src="https://unpkg.com/@browser-logos/firefox@2.0.0/firefox_16x16.png" width="16" height="16">Firefox (desktop and Android) and Chromium-based browsers including Chrome or Brave. Check out all of IPFS Companion's features and [**install it**](https://github.com/ipfs/ipfs-companion#install-ipfs-companion) today!
 
 ### IPFS and the JavaScript ecosystem
```
Comment (walkerlj0) [inline content/en/curriculum/ipfs/ipfs-gateways.md:97]:
```suggestion
### IPFS and JavaScript
```

File: content/en/curriculum/ipfs/ipfs-gateways.md [hunk 5]
```diff
@@ -52,131 +59,93 @@ Example:
...
     - W3C
     - IPFS and Igalia collaborate on dweb in browsers
     - Brave
     - Opera
-- Get involved!
-- Resources
-
-## Current Projects
 
 ### IPFS Companion browser extension
```
Comment (walkerlj0) [inline content/en/curriculum/ipfs/ipfs-gateways.md:93]:
```suggestion
### IPFS Companion Browser Extension
```

File: content/en/curriculum/ipfs/ipfs-gateways.md [hunk 6]
```diff
@@ -52,131 +59,93 @@ Example:
...
-* Work is in progress to improve Chromium’s consistency and specification compliance regarding the notion of [secure contexts](https://w3c.github.io/webappsec-secure-contexts/), including removing non-standard localhost names.
-* Miscellaneous other fixes have landed for the Firefox and Chromium’s implementations of custom handlers.
-* WIP refactor to make it easier to register custom protocol handlers (video: [Integrating New Protocol Handlers into Chrome [BlinkOn 15]](https://www.youtube.com/watch?v=kHIN6FkLAS8))
+#### IPFS and Igalia team up
 
+In 2020 IPFS and [Igalia](https://www.igalia.com/), an open source company with expertise in browser development and the web platform, started a collaboration that continues to this day. This opportunity has started discussions and has risen awareness of the distributed web. Their effort has brought much needed progress to bringing IPFS to Firefox and Chromium-based browsers. For more information [take a look at the official announcement](https://blog.ipfs.io/2021-01-15-ipfs-and-igalia-collaborate-on-dweb-in-browsers/).
 
 #### Brave
```
Comment (walkerlj0) [inline content/en/curriculum/ipfs/ipfs-gateways.md:135]:
```suggestion

```

Comment (walkerlj0) [reply content/en/curriculum/ipfs/ipfs-gateways.md:135]: Moving this up

File: content/en/curriculum/ipfs/ipfs-gateways.md [hunk 7]
```diff
@@ -52,131 +59,93 @@ Example:
...
 Diagnostic UI can be found at `brave://ipfs`, we suggest enabling IPFS Companion for the best experience
 
-TLDR integration status:
-
-- Initial release (v1.19) is focused on daemon orchestration and on URI support (read [blogs and press](https://github.com/ipfs/in-web-browsers/issues/64#issuecomment-763016248))
-- Demo: Opening `ipfs://{cid}` will trigger install prompt for go-ipfs managed by Brave itself.
-- For the best experience enable IPFS Companion and switch it to IPFS Node Type ["Provided by Brave"](https://docs.ipfs.io/how-to/companion-node-types/#provided-by-brave).
- When Companion is enabled all IPFS resources will be resolved by the local node.
-
 #### Opera
```
Comment (walkerlj0) [inline content/en/curriculum/ipfs/ipfs-gateways.md:140]:
```suggestion

```

Comment (walkerlj0) [reply content/en/curriculum/ipfs/ipfs-gateways.md:140]: Moved up

**Review by walkerlj0 — CHANGES_REQUESTED** [review]

File: content/en/curriculum/ipfs/ipfs-gateways.md [hunk 8]
```diff
@@ -52,131 +59,93 @@ Example:
 ipfs://bafybeiemxf5abjwjbikoz4mc3a3dla6ual3jsgpdr4cjr3oz3evfyavhwq/wiki/Vincent_van_Gogh.html
 ```
 
-
-
-> Native URI requires CID to be case-insensitive. Use of CIDv1 in Base32 is advised.
+> **Native URI requires the CID to be case-insensitive. Use of CIDv1 in Base32 is advised.**
```
[the remark this answers is not in the document]
Comment (mrodriguez3313) [reply content/en/curriculum/ipfs/ipfs-gateways.md:62]: @walkerlj0 Can you point me to the source for CIDv2? or did you mean CIDv0?

File: content/en/curriculum/ipfs/ipfs-gateways.md [hunk 9]
```diff
@@ -52,131 +59,93 @@ Example:
...
-**Gateway Recipes**
-
-_Read about [Gateway Recipes](https://github.com/ipfs/go-ipfs/blob/master/docs/config.md#gateway-recipes)_
-
-
-## IPFS In Web Browsers
+### Subdomain gateway
 
-<!-- What else should be included? Links? Can we pare it down?-->
+When [origin-based security](https://en.wikipedia.org/wiki/Same-origin_policy) is needed, CIDv1's case-insensitive encoding should be used in the subdomain:
```
[the remark this answers is not in the document]
Comment (mrodriguez3313) [reply content/en/curriculum/ipfs/ipfs-gateways.md:68]: This whole page is a deep page. See line 11. Would you prefer we give the deep residents less technical things to read through on this page then?

File: content/en/curriculum/ipfs/ipfs-gateways.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (walkerlj0) [reply content/en/curriculum/ipfs/ipfs-gateways.md:59]: I meant CIDv0, sorry! updated the suggestion

File: content/en/curriculum/ipfs/ipfs-gateways.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (walkerlj0) [reply content/en/curriculum/ipfs/ipfs-gateways.md:65]: I just think this part is unecessary, we can cut it, and just add something like 'learn more about read more about locators requiring origin-based security'

File: config/_default/goals.json [hunk 10]
```diff
@@ -64,6 +64,23 @@
...
+          "id": "1.31",
+          "description": "Know the methods to address content in IPFS through Gateways"
+        },
+        {
+          "id": "1.32",
+          "description": "Get familiar with the work being done to bring native IPFS support to browsers"
+        },
+        {
+          "id": "1.33",
+          "description": "Learn about current public Gateways and how any node in the network can become one"
```
Comment (walkerlj0) [inline config/_default/goals.json:80]:
```suggestion
          "description": "Learn about current public Gateways and how any node in the network can provide this service"
```

File: config/_default/goals.json [hunk 11]
```diff
@@ -64,6 +64,23 @@
         }
       ]
     },
+    "1.3":{
+      "description": "Learn about IPFS Gateways and how they bridge web2 and web3",
```
Comment (walkerlj0) [inline config/_default/goals.json:68]:
```suggestion
      "description": "Learn about IPFS Gateways and how they help bridge information between web2 and web3",
```

**Pull request merged**